sklearn/metrics/regression.py

```diff
@@ -33,7 +36,7 @@
 ]
 
 
-def _check_reg_targets(y_true, y_pred):
+def _check_reg_targets(y_true, y_pred, output_weights):
```

Comment:
Could you check whether it is acceptable to make `output_weights=None` a default? This is making Travis fail on `mean_absolute_error`.

It may be better to not set the default and be explicit in `mean_absolute_error`